Sources/Signal.swift

```diff
@@ -426,6 +426,12 @@ extension Signal: SignalProtocol {
 	}
 }
 
+extension Signal: SignalProducerRepresentable {
+	public var producer: SignalProducer<Value, Error> {
+		return SignalProducer(self)
```

Comment:
Lifting a `Signal` to `SignalProducer` does add an extra relaying signal. But there are ways to mitigate the overhead.